### ISSUE TITLE
Regarding transforms

### ISSUE THREAD

**Issue by SidharthAgarwal18** [ain-soph/trojanzoo#64]
Can somebody please let me know the transforms used in cfir10 and mnist datasets?
Thank you.

**Comment by ain-soph** [comment]
https://github.com/ain-soph/trojanzoo/blob/master/trojanvision/datasets/imageset.py#L56-L61

**Comment by ain-soph** [comment]
```
CUDA_VISIBLE_DEVICES=0 python examples/train.py --verbose 1 --color --epoch 600 --batch_size 96 --cutout --grad_clip 5.0 --lr 0.025 --lr_scheduler --save --dataset cifar10 --model resnet18_comp
```
And you will get ResNet18 (first convolutional layer compressed version) with 96.5% accuracy.

**Comment by SidharthAgarwal18** [comment]
ya so I was testing these models manually and the transform that worked for me is

transform.Compose([transforms.ToTensor(),transforms.Normalize([0.49139968, 0.48215827, 0.44653124],[0.24703233, 0.24348505, 0.26158768]))
To give a good accuracy.

**Comment by ain-soph** [comment]
Close this issue if you have no further concern.
